code/modules/fishing/fish/fish_traits.dm

```diff
@@ -226,6 +303,10 @@ GLOBAL_LIST_INIT(fish_traits, init_subtypes_w_path_keys(/datum/fish_trait, list(
 	else
 		source.visible_message(message)
 
+/datum/fish_trait/revival/apply_to_mob(mob/living/basic/mob)
+	. = ..()
+	mob.AddComponent(/datum/component/regenerator, regeneration_delay = 6 SECONDS, brute_per_second = 2 SECONDS, outline_colour = COLOR_BLUE)
```

Comment:
Probably sensible for it to be weaker on a mob than a fish, but a shame 😅

Reply:
I think it's laziness that's stopped me from making it revive mobs, but I could as well do it later though, I'm just going to be a bit busy tomorrow.